include/oneapi/tbb/detail/_flow_graph_impl.h

```diff
@@ -432,7 +454,7 @@ inline graph_task* prioritize_task(graph& g, graph_task& gt) {
     //! priority queue, and a new critical task is created to take and execute a work item with
     //! the highest known priority. The reference counting responsibility is transferred (via
     //! allocate_continuation) to the new task.
```

Comment:
The comment seems to be outdated.
```suggestion
    //! the highest known priority. The reference counting responsibility is transferred to
    //! the new task.
```

Reply:
Removed